components/PlantCharacteristics/index.js

```diff
@@ -26,7 +26,7 @@ const StyledCharacteristic = styled.div`
   align-items: center;
 `;
 const StyledCharacterHeadline = styled.h3`
-  font-size: 1rem;
+  font-size: 0.9rem;
   margin: 0;
 `;
 
```

Comment:
The wrapper for your component is a `div` element, but when you use `figcaption` it's parent should be a `figure` element[(see Caninclude)](https://caninclude.glitch.me/caninclude?child=figcaption&parent=div). Something you can use here. :)